CHANGELOG.md

```diff
@@ -1,5 +1,25 @@
 # Braintree Android SDK Release Notes
 
+## unreleased
+
+* Local Payment
+  * Add required property `hasUserLocationConsent` to `LocalPaymentRequest`
```

Comment:
It's not a required property in this major version though right? That would be a breaking change

Reply:
Ah yeah, it's not a required property since the old constructor still exists. I'll update the wording here.